src/napari_threedee/manipulators/base_manipulator.py

```diff
@@ -67,6 +68,69 @@ def origin(self) -> np.ndarray:
     def origin(self, value: np.ndarray):
         self._backend.manipulator_model.origin = value
 
+    @property
+    def radius(self) -> float:
+        """The radius of the manipulator components.
+        """
+        return self._backend.manipulator_model.radius
```

Comment:
Do I just add a self._radius and init it to 20 and then set it in the setter and return in the getter?
This would work, would just be another place to edit if changing the defaults (vs. the values in the implementation classes).
I'm a bit out of my depth here i think. cc: @kevinyamauchi

Reply:
I implemented it in this fashion for now, just to see how it would look and work. 
Seems ok?